src/Symlinker.php

```diff
@@ -96,7 +96,7 @@ private function link($sourcePath, $destinationPath, IOInterface $io)
                 $io->write(': <comment>symlink created</comment>', true, IOInterface::VERBOSE);
             }
         } catch (IOException $e) {
-            throw new \RuntimeException(sprintf('   Symlink from "%s" to "%s" failed!', $sourcePath, $destinationPath));
+            throw new \RuntimeException(sprintf('   Symlink from "%s" to "%s" failed! Error: %3', $sourcePath, $destinationPath, $e->getMessage()));
```

Comment:
Please, change the placeholder to `%s` and add the `IOException` as the third argument of the `RuntimeException` constructor
```suggestion
            throw new \RuntimeException(sprintf('   Symlink from "%s" to "%s" failed! Error: "%s"', $sourcePath, $destinationPath, $e->getMessage()), 0, $e);
```